pkg/core/native/native_neo.go

```diff
@@ -638,7 +650,7 @@ func (n *NEO) calculateBonus(d dao.DAO, vote *keys.PublicKey, value *big.Int, st
 	}
 
 	var key = makeVoterKey(vote.Bytes())
-	var reward = n.getGASPerVote(d, key, start, end)
+	var reward = n.getGASPerVote(d, key, []uint32{start, end})
 	var tmp = new(big.Int).Sub(&reward[1], &reward[0])
```

Comment:
And we can probably save on allocation here.

Reply:
Fixed.